dev/ci/gitlab-modes/tagged-runners.yml

```diff
@@ -1,3 +1,11 @@
 .auto-use-tags:
   tags:
     - $TAGGED_RUNNERS
+
+# making this configurable is too annoying if we want to support > 1 tag
+# and not sure small is enough
+# so just hardcode ci.inria.fr && medium
+.auto-use-docker-tags:
+  tags:
+    - ci.inria.fr
+    - medium
```

Comment:
Did we check `medium` will work too? I see the last test was done with `large`

Reply:
Ping @SkySkimmer

Reply:
Yeah, that got me wondering why you reverted to `medium`, wouldn't it better to run the jobs on `large` ?

Reply:
It doesn't seem to be faster so why not medium

Reply:
I see, thanks, I just wanted to be sure the medium change was intentional, I didn't realize you had bumped the timeout too.